src/andromede/main/main.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+import argparse
+from pathlib import Path
+from typing import Optional
+
+from andromede.model.library import Library
+from andromede.model.parsing import parse_yaml_library
+from andromede.model.resolve_library import resolve_library
+from andromede.simulation import OutputValues, TimeBlock, build_problem
+from andromede.study import DataBase
+from andromede.study.parsing import parse_yaml_components
+from andromede.study.resolve_components import (
+    NetworkComponents,
+    build_data_base,
+    build_network,
+    consistency_check,
+    resolve_components_and_cnx,
+)
+
+
+class AntaresTimeSeriesImportError(Exception):
+    pass
+
+
+def input_models(model_path: Path) -> Library:
+    with model_path.open() as lib:
+        return resolve_library(parse_yaml_library(lib))
+
+
+def input_database(study_path: Path, timeseries_path: Optional[Path]) -> DataBase:
+    with study_path.open() as comp:
+        return build_data_base(parse_yaml_components(comp), timeseries_path)
+
+
+def input_components(study_path: Path, model: Library) -> NetworkComponents:
+    with study_path.open() as comp:
+        return resolve_components_and_cnx(parse_yaml_components(comp), model)
+
+
+if __name__ == "__main__":
```

Comment:
For clarity, I suggest to put all that is bellow here in a "main" function:
```suggestion
if __name__ == "__main__":
  main()
```